database/sql/src/main/scala/cromwell/database/sql/CallCachingStore.scala

```diff
@@ -1,18 +1,20 @@
 package cromwell.database.sql
 
-import cromwell.database.sql.tables.{CallCachingHashEntry, CallCachingResultMetaInfoEntry, CallCachingResultSimpletonEntry}
+import cromwell.database.sql.tables.{CallCachingJobDetritusEntry, CallCachingHashEntry, CallCachingResultMetaInfoEntry, CallCachingResultSimpletonEntry}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 trait CallCachingStore {
   def addToCache(callCachingResultMetaInfo: CallCachingResultMetaInfoEntry,
                  hashesToInsert: Int => Iterable[CallCachingHashEntry],
-                 resultToInsert: Int => Iterable[CallCachingResultSimpletonEntry])
+                 resultToInsert: Int => Iterable[CallCachingResultSimpletonEntry],
+                 jobDetritusToInsert: Int => Iterable[CallCachingJobDetritusEntry])
                 (implicit ec: ExecutionContext): Future[Unit]
 
   def metaInfoIdsMatchingHashes(hashKeyValuePairs: Seq[(String, String)])
                                (implicit ec: ExecutionContext): Future[Seq[Seq[Int]]]
 
   def fetchCachedResult(callCachingResultMetaInfoId: Int)(implicit ec: ExecutionContext):
-  Future[(Option[CallCachingResultMetaInfoEntry], Seq[CallCachingResultSimpletonEntry])]
+                         Future[(Option[CallCachingResultMetaInfoEntry],
+                         Seq[CallCachingResultSimpletonEntry], Seq[CallCachingJobDetritusEntry])]
```

Comment:
And if one means `type` and not `case class`, they're wrong and should see the previous comment ;)